ansible/roles/openondemand/tasks/vnc_compute.yml

```diff
@@ -27,13 +27,15 @@
   # Requires separate step so that the upgraded pip is used to install packages
   pip:
     name: pip
+    state: 23.2.1
```

Comment:
Passing a version to `state` is [not documented](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html#parameter-state) as working